src/Accordion.php

```diff
@@ -43,19 +39,14 @@
  *             'options' => [...],
  *             'footer' => 'Footer' // the footer label in list-group
  *         ],
- *     ]
- * ]);
+ *     ]);
  * ```
- *
- * @see https://getbootstrap.com/docs/4.2/components/collapse/#accordion-example
- * @author Antonio Ramirez <amigo.cobos@gmail.com>
- * @author Simon Karlen <simi.albi@gmail.com>
  */
 class Accordion extends Widget
 {
     /**
-     * @var array list of groups in the collapse widget. Each array element represents a single
-     * group with the following structure:
+     * @var array list of groups in the collapse widget. Each array element represents a single group with the following
+     *            structure:
```

Comment:
```suggestion
     * structure:
```

Reply:
Do we have indents in all the packages in which the variables are defined, are they going to be removed?

Reply:
Yes. We're not using indents.